geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/server/CommandIntegrationTest.java

```diff
@@ -19,20 +19,23 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
 import io.lettuce.core.RedisClient;
 import io.lettuce.core.api.sync.RedisCommands;
+import junit.framework.AssertionFailedError;
```

Comment:
You should use `java.lang.AssertionError` instead. `junit.framework` is JUnit 1-1.3, while `org.junit` is JUnit 1.4. Anything in `junit.framework` should be considered deprecated.

Reply:
Done.